packages/amplify-e2e-core/src/utils/rds.ts

```diff
@@ -410,6 +410,17 @@ export const deleteDbConnectionConfig = async (options: {
   await ssmClient.send(new DeleteParametersCommand(input));
 };
 
+export const deleteDbConnectionStringConfig = async (options: { region: string; connectionUriSsmPath: string }): Promise<void> => {
+  const ssmClient = new SSMClient({ region: options.region });
+
+  const input: DeleteParametersCommandInput = {
+    Names: [options.connectionUriSsmPath],
+  };
+
+  console.log('Deleting SSM parameters');
+  await ssmClient.send(new DeleteParametersCommand(input));
+};
+
```

Comment:
Can we make it generic like `deleteSSMParameter` rather than creating a new method for each attribute?